packaging/install/040-Crd-kafkaaccess.yaml

```diff
@@ -11,57 +11,57 @@ spec:
     kind: KafkaAccess
     plural: kafkaaccesses
     shortNames:
-    - ka
+      - ka
```

Comment:
Why do we have the changes here in this file?

Reply:
The `Makefile`s `crd_install` command overrides this. `make build` or  `make all` uses that command.

Reply:
Yeah, but why did the files changed?

Reply:
I guess its environmental. Let me check.

Reply:
The crd_install in makefile uses yq. yq currently has some issues with preserving the indentation (there are many open issues for both version 3 and 4). 

I tried to enforce an indentation level and seems that it is buggy as well. Check out this issue: https://github.com/mikefarah/yq/issues/825

So options here are:

a- We can keep the indentation as generated by yq as we cannot enforce it. The YAML format doesn't change, just the indentation level.
b- we can keep the old file format. But this will need a manual effort each time we run `make build` or `make all`. Kate probably did the same as she proposed using yq 4.2.1+. I tested it many versions starting yq 4.2.1 and still the same result.

Do you have any suggestions?

Reply:
Please also note that the indentation level of this YAML file is the same as the other YAMLs in the same packaging directory. I can revert it, but each time we run the make command, it will change the indentation back to this one.

It is either Kate did a trick with this to skip the regeneration or she has a different configuration which I am not sure of.